### PR TITLE
documentation: ZENKO-2409_install_guide_walkthru_changes

### PR DESCRIPTION
**Pull request by wabernat**
Made changes recommended by TS walkthrough of Zenko install guide.

**What does this PR do, and why do we need it?**

This PR implements most of XF's walkthrough of MK8s/Zenko instructions. Expect further changes  as a result of my walkthrough in ZENKO-2421.

**Which issue does this PR fix?**

fixes #ZENKO-2409

### CODE REVIEW

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 1]
```diff
@@ -63,18 +63,20 @@ To upgrade Zenko:
...
+   ::
+      
       $ helm upgrade {{zenko-server-name}} ./zenko
 
    If you are using custom values, reuse options.yaml on upgrades.
+   
    ::
 
-      $ helm upgrade {{zenko-server-name}} ./zenko -f options.yaml
+      $ helm upgrade {{zenko-server-name}} . -f options.yaml
```
Comment (jonathan-gramain) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]: I think this line is correct and should not be changed.

Comment (wabernatScality) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: I agree. Will roll it back. Thanks.

Comment (wabernat) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: Double-checking, I see the folllowing text at: installation/configure/configuring_zenko.html

"The options.yaml file is not present by default, but you added a simple one at Zenko/kubernetes/options.yaml when deploying Zenko."

The problem here is that in 1.1, we recommended Zenko/kubernetes/zenko/options.yaml. 
In 1.2 we're recommending a new structure: Zenko/kubernetes/options.yaml. 

Xavier caught a lot of these. Please review this to make sure I'm not crazy.

Comment (jonathan-gramain) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: I agree with this, though the line as it is now looks correct, right? So starting from the `kubernetes` directory, we are upgrading the `./zenko` directory, using `./options.yaml`, which looks correct to me.

Comment (jonathan-gramain) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: Also you can compare with the `helm upgrade` line above at line 68.

Comment (wabernat) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: Okay, that's persuasive. I am making the requested change. Sorry about the confusion.

**Review by jonathan-gramain — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e approve

**Review by dora-korpar — APPROVED** [review]

File: docs/docsource/installation/prepare/sizing.rst [hunk 2]
```diff
@@ -38,37 +39,39 @@ Reserve the following resources for each node.
...
+   demand. Once set, the cluster cannot be resized without redefining new
+   volumes.
 
-To size your deployment, review the default values in Zenko/kubernetes/zenko/\
-values.yaml. This file reserves space for each component
-in the build. This is the baseline build, which Helm will install unless
-instructed otherwise.
+To size your deployment, review the default values in
+Zenko/kubernetes/zenko/values.yaml. This file reserves space for each component
+in the build. values.yaml contains baseline build sesttings, which Helm installs
```
Comment (rahulreddy) [inline docs/docsource/installation/prepare/sizing.rst]: ~sesttings~ settings

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**